app/assets/stylesheets/views/_homepage.scss

```diff
@@ -251,3 +251,17 @@
     border-top: 4px solid $govuk-brand-colour;
   }
 }
+
+// Temporary overrides for global bar styling on homepage.
+// Doing this here avoids the need for republishing the
+// gem and therefore will be easier to remove later.
+.homepage.global-bar-present .global-bar {
+  border: 0;
+}
+
+.homepage.global-bar-present .homepage-header {
```

Comment:
Styles for the `.homepage-header` class are in https://github.com/alphagov/frontend/blob/af433e2d0b937964023ee8870b0d8b926c6fe071/app/assets/stylesheets/views/_homepage_header.scss#L5 - why have they been added here?

Reply:
I reasoned that it made sense to keep the lines in this commit together in the `_homepage.scss` file to identify them as related and to enable quick removal when they are no longer needed (as all this will be refactored). As we need the specific `.homepage.global-bar-present` classes to hook into, I reasoned that these rules can live in the `_homepage.scss` since that's the logical 'parent' SCSS file for this piece of work.

Happy to move them if you think it's necessary, but I'd prefer to keep these together for the reasons given above.

Reply:
Okay, that's reasonable.